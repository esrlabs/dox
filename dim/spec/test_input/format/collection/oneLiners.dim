document: one_liners

REQ_0a: info In total 150 chars in one-liner......................................................................................
REQ_0b:
    type: information
    text: In total 150 chars in one-liner......................................................................................
    tags:
    comment:

REQ_0c: info In total 151 chars in one-liner.......................................................................................
REQ_0d:
    type: information
    text: In total 150 chars in one-liner......................................................................................

REQ_1a:   |
          info
          Single Line Stripped

REQ_1b: |
        info
        Multi Line:
        - bla
        - fasel

REQ_2a:
    type: heading_0
    text: MyHeading
