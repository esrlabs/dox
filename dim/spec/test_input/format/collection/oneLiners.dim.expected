document: one_liners

REQ_0a:                     info In total 150 chars in one-liner......................................................................................

REQ_0b:
  type: information
  text:                     In total 150 chars in
                            one-liner......................................................................................
  tags:
  comment:

REQ_0c:
  type: information
  text:                     In total 151 chars in
                            one-liner.......................................................................................

REQ_0d:                     info In total 150 chars in one-liner......................................................................................

REQ_1a:                     info Single Line Stripped

REQ_1b:
  type: information
  text:                     |
                            Multi Line:
                            - bla
                            - fasel

REQ_2a:                     h0 MyHeading
